client/windows/CSpellWindow.cpp

```diff
@@ -449,9 +444,13 @@ void CSpellWindow::keyPressed(const SDL_KeyboardEvent & key)
 		case SDLK_DOWN:
 		{
 			bool down = key.keysym.sym == SDLK_DOWN;
-			static const int schoolsOrder[] = { 0, 3, 1, 2, 4 };
+			static const int schoolsOrder[] =
+			{
+				0, 3, 1, 2, 4
+			};
 			int index = -1;
-			while(schoolsOrder[++index] != selectedTab);
+			while(schoolsOrder[++index] != selectedTab)
+				;

```

Comment:
This is something uncrustify doesn't handle well, but this probably need to be rewritten anyway.